cmake/CMakePackageConfigHelpers.cmake

```diff
@@ -0,0 +1,227 @@
+# - CONFIGURE_PACKAGE_CONFIG_FILE(), WRITE_BASIC_PACKAGE_VERSION_FILE()
```

Comment:
This file is part of [CMake modules](https://cmake.org/cmake/help/latest/module/CMakePackageConfigHelpers.html), why do you need a copy here ?